src/apis/axios/index.ts

```diff
@@ -0,0 +1,19 @@
+import axios, { AxiosRequestConfig } from 'axios';
+
+const DEFAULT_TIMEOUT = 30000;
+
+export const createClient = (config?: AxiosRequestConfig) => {
+  const axiosInstance = axios.create({
+    baseURL: process.env.VITE_SERVER_URL,
+    timeout: DEFAULT_TIMEOUT,
+    headers: {
+      'content-type': 'application/json',
+    },
+    withCredentials: true,
+    ...config,
+  });
+
+  return axiosInstance;
+};
+
+export const axiosClient = createClient();
```

Comment:
config 값은 객체 리터럴에 직접 넣어주고 이렇게 표현해줘도 되지 않나요?
```suggestion
  export const axiosClient = axios.create({
    baseURL: process.env.VITE_SERVER_URL,
    timeout: DEFAULT_TIMEOUT,
    headers: {
      'content-type': 'application/json',
    },
    withCredentials: true,
    ...config,
  });
```